apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Footer/HostSidePanel/HostSidePanel.tsx

```diff
@@ -60,22 +89,39 @@ export function HostSidePanel(): ReactElement {
   }
 
   const handleSelectHost = async (hostId: string): Promise<void> => {
-    setSelectedHost(data.hosts.find((host) => host.id === hostId))
+    setSelectedHost(teamHosts.hosts.find((host) => host.id === hostId))
     setOpenSelect(false)
   }
 
   return (
     <>
       {/* DefaultHostPanel - no host */}
       <SidePanel title={t('Hosted By')} open edit>
-        {!openSelect && selectedHost == null && (
-          <SidePanelContainer border={false}>
-            <ContainedIconButton
-              label={t('Select a Host')}
-              thumbnailIcon={<UserProfiles2Icon />}
-              onClick={() => setOpenSelect(true)}
-            />
-          </SidePanelContainer>
+        {!openSelect && (selectedHost == null || !userInTeam) && (
+          <>
+            <SidePanelContainer border={!userInTeam}>
```

Comment:
This should stay like this right? Is there a circumstance when it's true?
```suggestion
            <SidePanelContainer border={false}>
```

Reply:
If the user isn't in the team, we show the error message - and then we need to add a border between the error message and the button.